index.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     {
       files: ['**/*.js', '**/*.jsx'],
       rules: {
+        '@typescript-eslint/no-unsafe-argument': 'off',
```

Comment:
[@typescript-eslint/no-unsafe-argument](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unsafe-argument.md) is a new recommended rule that is likely to result in a bunch of linting errors across existing projects. For reference, skuba itself required the following tweaks: https://github.com/seek-oss/skuba/pull/806/commits/b887ccf39d6a41f9a002c153200630f54d20bc4a

Currently, we're only disabling it for JavaScript source files and tests. Should we be less aggressive across the board by e.g. defaulting it to `warn` instead of `error` to begin with? The counterpoint is that warnings are often left ignored, which is why we tend to avoid `warn`s elsewhere in the config.

Reply:
I wonder if you have an opinion here @samchungy, I noticed the Orders API has some warnings 😅

Reply:
It seems like a good rule but I can see how it would be noisy on an existing code base. I'm leaning towards leaving it on and trusting upstream.

Reply:
Hahaha don't look at that repo... 🙈 Funnily enough it's not even the most pressing issue with that repo 😭.

My plan was to leave those warnings on until it annoys enough people for us to allocate time to it 😂. Leave it on is my opinion too. Worst case they can turn it off